### PR TITLE
Amazon EC2 2.4.0

### PR DESCRIPTION
**Pull request by dhirsch1**

### CODE REVIEW

**Review by devdanco — DISMISSED** [review]

File: amazon-ec2/2.4/modules/ROOT/pages/amazon-ec2-connector-examples.adoc [hunk 1]
```diff
@@ -0,0 +1,214 @@
+= Amazon EC2 Connector 2.4 Example - Mule 4
+:page-aliases: connectors::amazon/amazon-ec2-connector-examples.adoc
+
+This example creates an Amazon EBS (Elastic Block Store) volume that you can attach to an Amazon EC2 instance that is in the same availability zone as the volume.
+
+The example includes configuring global elements for Anypoint for Amazon EC2 (Amazon EC2 Connector), configuring additional properties such as volume size and type, configuring the *Set Payload* component, adding and configuring the *Logger* component, and testing your application's use of the connector.
```
Comment (kevintroller) [inline amazon-ec2/2.4/modules/ROOT/pages/amazon-ec2-connector-examples.adoc]:
```suggestion
The example includes configuring global elements for Anypoint Connector for Amazon EC2 (Amazon EC2 Connector), configuring additional properties such as volume size and type, configuring the *Set Payload* component, adding and configuring the *Logger* component, and testing your application's use of the connector.
```

File: amazon-ec2/2.4/modules/ROOT/pages/amazon-ec2-connector-examples.adoc [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+This prints the volume ID of the EBS volume.
++
+The following image shows the *Set Payload* configuration:
++
+image::amazon-ec2-create-volume-payload.png[EC2 Create Volume payload transformer]
+
+== Add and Configure the Logger Component
+
+. In the *Mule Palette* view, select *Logger* and drag it to the right of the *Set Payload* component. +
+The *Logger* component displays the volume ID that is transformed by *Set Payload* from the *Create Volume* operation in the Mule console.
```
Comment (kevintroller) [inline amazon-ec2/2.4/modules/ROOT/pages/amazon-ec2-connector-examples.adoc]:
```suggestion
The *Logger* component displays the volume ID that is transformed by the *Set Payload* component from the *Create Volume* operation in the Mule console.
```

File: amazon-ec2/2.4/modules/ROOT/pages/amazon-ec2-connector-studio.adoc [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+* *Scheduler* +
+Initiates a flow when a time-based condition is met
+
+For example, to configure an HTTP listener, follow these steps:
+
+. In the *Mule Palette* view, select *HTTP > Listener*.
+. Drag *Listener* to the Studio canvas.
+. On the *Listener* configuration screen, optionally change the value of the *Display Name* field.
+. Specify a value for the *Path* field.
+. Click the plus sign (+) next to the *Connector configuration* field to configure a global element that can be used by all instances of HTTP *Listener* in the app.
```
Comment (kevintroller) [inline amazon-ec2/2.4/modules/ROOT/pages/amazon-ec2-connector-studio.adoc]:
```suggestion
. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of HTTP *Listener* in the app.
```

File: amazon-ec2/2.4/modules/ROOT/pages/amazon-ec2-connector-reference.adoc [hunk 4]
```diff
@@ -0,0 +1,5034 @@
...
+
+See <<exceptionthrows1>>
+
+[[copySnapshot]]
+=== Copy Snapshot
+`<ec2:copy-snapshot>`
+
+Copies a point-in-time snapshot of an EBS volume and stores it in Amazon S3. You can copy the snapshot within the same region, or from one region to another. You can use the snapshot to create EBS volumes or Amazon Machine Images (AMIs).
+
+Copies of encrypted EBS snapshots remain encrypted and copies of unencrypted snapshots remain unencrypted. To encrypt unencrypted snapshots, specify thh `Encrypted` during the copy snapshot operation. Encrypted snapshot copies use the default AWS Key Management Service (AWS KMS) customer master key (CMK). You can use the `KmsKeyId` parameter to specify a non-default CMK. To copy an encrypted snapshot that is shared from another account, you must have permissions for the CMK that is used to encrypt the snapshot.
```
Comment (kevintroller) [inline amazon-ec2/2.4/modules/ROOT/pages/amazon-ec2-connector-reference.adoc]:
```suggestion
Copies of encrypted EBS snapshots remain encrypted and copies of unencrypted snapshots remain unencrypted. To encrypt unencrypted snapshots, specify the `Encrypted` parameter during the copy snapshot operation. Encrypted snapshot copies use the default AWS Key Management Service (AWS KMS) customer master key (CMK). You can use the `KmsKeyId` parameter to specify a non-default CMK. To copy an encrypted snapshot that is shared from another account, you must have permissions for the CMK that is used to encrypt the snapshot.
```

File: amazon-ec2/2.4/modules/ROOT/pages/amazon-ec2-connector-reference.adoc [hunk 5]
```diff
@@ -0,0 +1,5034 @@
...
+
+[[describeSnapshots]]
+=== Describe Snapshots
+`<ec2:describe-snapshots>`
+
+Describes one or more of the EBS snapshots available to you. Available snapshots include:
+
+* Public snapshots available for any AWS account to launch
+* Private snapshots that you own
+* Private snapshots owned by another AWS account but for which you have explicit create volume permissions.
```
Comment (kevintroller) [inline amazon-ec2/2.4/modules/ROOT/pages/amazon-ec2-connector-reference.adoc]:
```suggestion
* Private snapshots owned by another AWS account but for which you have explicit create volume permissions
```

File: amazon-ec2/2.4/modules/ROOT/pages/amazon-ec2-connector-reference.adoc [hunk 6]
```diff
@@ -0,0 +1,5034 @@
...
+
+See <<exceptionthrows1>>
+
+[[reportInstanceStatus]]
+=== Report Instance Status
+`<ec2:report-instance-status>`
+
+Submits feedback about the status of an instance. The instance must be in the running state.
+
+If your experience with the instance differs from the instance status returned by DescribeInstanceStatus, use ReportInstanceStatus to report your experience with the instance. Amazon EC2 collects this information to improve the accuracy of status checks. Use of this action does not change the value returned by DescribeInstanceStatus.
```
Comment (kevintroller) [inline amazon-ec2/2.4/modules/ROOT/pages/amazon-ec2-connector-reference.adoc]:
```suggestion
If your experience with the instance differs from the instance status returned by `DescribeInstanceStatus`, use `ReportInstanceStatus` to report your experience with the instance. Amazon EC2 collects this information to improve the accuracy of status checks. Use of this action does not change the value returned by DescribeInstanceStatus.
```

**Review by kevintroller — DISMISSED** [review]
Approved, minor comments

**Review by devdanco — APPROVED** [review]

**Review by kevintroller — APPROVED** [review]

**Pull request merged**